### DISCUSSION TITLE
attach sqlite db, what engine does the work?

### DISCUSSION THREAD

**Discussion by Moaneschien** [duckdb/duckdb#15369] [Q&A] [answered]
When I attach a SQLite DB to DuckDB and run a query against that DB. What engine does do the actual work?

**Comment by szarnyasg** [comment, marked as answer]
Hi @Moaneschien, great question. In these cases, DuckDB scans the SQLite database file directly and DuckDB's engine computes the query results.

**Reply by Moaneschien** [reply]
Thanks.
